vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -296,21 +334,21 @@ private List getContainerList(config) {
 
     //If no custom jnlp agent provided as default jnlp agent (jenkins/jnlp-slave) as defined in the plugin, see https://github.com/jenkinsci/kubernetes-plugin#pipeline-support
     def result = []
-
     //allow definition of jnlp image via environment variable JENKINS_JNLP_IMAGE in the Kubernetes landscape or via config as fallback
     if (env.JENKINS_JNLP_IMAGE || config.jenkinsKubernetes.jnlpAgent) {
         result.push([
-            name: 'jnlp',
+            name : 'jnlp',
             image: env.JENKINS_JNLP_IMAGE ?: config.jenkinsKubernetes.jnlpAgent
         ])
     }
     config.containerMap.each { imageName, containerName ->
         def containerPullImage = config.containerPullImageFlags?.get(imageName)
+        boolean pullImage = containerPullImage != null ? containerPullImage : config.dockerPullImage // dockerPullImage is per default set to true
```

Comment:
I think we do not need the comment. The developer should look up the default in the corresponding file. Could likely be that the default is changed, but the documentation here is not updated...

Reply:
done